runhouse/main.py

```diff
@@ -391,12 +392,50 @@ def _print_envs_info(
             for resource in resources_in_env:
                 for resource_name, resource_info in resource.items():
                     resource_type = _adjust_resource_type(
-                        resource_info["resource_type"]
-                    )
-                    console.print(
-                        f"{DOUBLE_SPACE_UNICODE}{BULLET_UNICODE} {resource_name} ({resource_type})"
+                        resource_info.get("resource_type")
                     )
 
+                    active_function_calls = resource_info.get("active_function_calls")
+                    resource_info_str = f"{DOUBLE_SPACE_UNICODE}{BULLET_UNICODE} {resource_name} ({resource_type})"
+
+                    if resource_type == "runhouse.Function":
+                        func_start_time_utc = (
+                            active_function_calls[0].get("start_time", None)
+                            if active_function_calls
+                            else None
+                        )
+
+                        # casting func_start_time_utc to datetime format
```

Comment:
nit but can we split this by whether the function is active for readability, instead of handling it separately for each computation?

```
if isinstance(resource_type, Function) and if resource_info.get("active_function_calls"):
    # get start + end time
    # compute running time
    # is func running str = ...
else:
    # is func running str = ...
full_str = ...
```

Reply:
@carolineechen we can't really execute `isinstance(resource_type, Function)`, because the `resource_type` is saved as str in the status json output, therefore I compared it to `"runhouse.Function"`.